models/public/colorization-v2/colorization-v2.md

```diff
@@ -0,0 +1,81 @@
+# colorization-v2
+
+## Use Case and High-Level Description
+
+The `colorization-v2` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+group of models designed to perform image colorization. For details
+about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+The gray-scale or BGR image is first converted to LAB color space. 
+The model input is a blob that consists of a L-channel of image of 1x1x224x224. 
+The mean values need to be subtracted as follows: 50.0
+before passing the image blob into the network. In addition, values must be divided
+by 0.0039.
```

Comment:
According to the demo, the image should be scaled at the very beginning, but not in the end of preprocessing. Moreover its values must be divided by 255, but not (1/255). The same goes for other descriptions.

Reply:
@Wovchena corrected

Reply:
It still reads like you need to subtract mean at first and than rescale. How about:
```
The gray-scale or BGR image should be preprocessed first:
1) divide image's values by 255
2) convert the image to LAB
3) extract l component with 50.0 subtracted from each value
```

Reply:
done